python/cudf/cudf/fuzz_tests/fuzz_test_parquet.py

```diff
@@ -19,6 +20,34 @@ def parquet_reader_test(parquet_buffer):
     assert_eq(gdf, pdf)
 
 
+@pythonfuzz(
+    data_handle=ParquetReader,
+    params={
+        "columns": None,
+        "engine": ["pyarrow", None],
+        "use_pandas_metadata": [True, False],
+    },
+)
+def parquet_reader_columns_engine(
+    parquet_buffer, columns, engine, use_pandas_metadata
+):
+    print("PRAMS", columns, engine, use_pandas_metadata)
```

Comment:
Is it possible to move this `print` to the `@pythonfuzz` decorator? That way we don't have to have a `print` at the beginning of every fuzz test.

Reply:
This was a debug statement added that slipped through. Added a logging statement for this in the core fuzzer instead.